lib/pages/new_group/new_group.dart

```diff
@@ -198,17 +213,16 @@ class NewGroupController extends State<NewGroup>
 
   void _handleUploadAvatarNewGroupChatOnData(BuildContext context, Either<Failure, Success> event) {
     Logs().d('NewGroupController::_handleUploadAvatarNewGroupChatOnData()');
-    uploadAvatarNewGroupChatNotifier.value = event;
+    createRoomStateNotifier.value = event;
```

Comment:
for me, we need to update the `stateEvent` not only in success, but also in error

Reply:
Handled error